CMakeLists.txt

```diff
@@ -78,7 +78,7 @@ endif()
 
 include(Flake8)
 
-set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD 20)
```

Comment:
Because I have used designate initialiser, this feature is supported by c++20, therefore change it to c++ standard 20.

Reply:
Is there a reason that you have to use designated initializer?  Please let us know what it is.

If there is not, then just don't use designated initializer.  I have no problem using it, but it alone is not a good reason to upgrade to C++20.

Reply:
I just want to mark the initial value for each structure member during structure initialization, but I did not consider what side effects upgrading to C++20 may bring.